### PR TITLE
[DE-505](analytics) feat: improve seed logic + locations

### PR DESCRIPTION
**Pull request by cdelabre**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/macros/generate_seed_geolocation_query.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% macro generate_seed_geolocation_query(source_table, referential_table, id_column, prefix_name, columns, geo_shape='geo_shape') %}
```
Comment (LucileRainteau) [inline orchestration/dags/data_gcp_dbt/macros/generate_seed_geolocation_query.sql:1]: ça serait utile de documenter cette macro dans _orchestration/dags/data_gcp_dbt/macros/schema.yml_ je pense

File: orchestration/dags/data_gcp_dbt/models/intermediate/seed/schema/int_seed__intercommunal_public_institution.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+version: 2
```
Comment (LucileRainteau) [inline orchestration/dags/data_gcp_dbt/models/intermediate/seed/schema/int_seed__intercommunal_public_institution.yml:1]: <3

File: orchestration/dags/data_gcp_dbt/models/intermediate/api_gouv/address/int_api_gouv__user_address.sql [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    adu.user_id,
+    adu.user_address,
+    adu.user_postal_code,
+    adu.user_department_code,
+    ul.longitude as user_longitude,
+    ul.latitude as user_latitude,
+    ul.city_code,
+    ul.api_adresse_city,
+    ul.date_updated
+from {{ source("raw", "user_locations") }} ul
```
Comment (LucileRainteau) [inline orchestration/dags/data_gcp_dbt/models/intermediate/api_gouv/address/int_api_gouv__user_address.sql:13]: Ici, les adresses des jeunes ne sont pas mises à jour en cas de changement d'adresse
(cf point relevé par Matthieu => [ticket ici ](https://www.notion.so/passcultureapp/880351701699470cb129e4e0ad6bbcba?v=f77024eeaeb14713a3f1c75b31d8aa80&p=d56337d8624b405190eefebac1b64a36&pm=s))

File: orchestration/dags/data_gcp_dbt/macros/generate_seed_geolocation_query.sql (hunk 1 above)
Comment (cdelabre) [reply orchestration/dags/data_gcp_dbt/macros/generate_seed_geolocation_query.sql:1]: Done !

**Pull request merged**